spec/src/main/asciidoc/architecture.adoc

```diff
@@ -53,9 +53,9 @@ The metrics can be hard coded into the server or read from a configuration file
 The Appendix shows a possible data format for such a configuration.
 The configuration and set up of the _base_ scope is thus an implementation detail and is not expected to be portable across vendors.
 
-Section <<required-metrics#required-metrics>> lists the required metrics. This list also includes a few items marked as optional.
-These are listed here as they are dependent on the underlying JVM and not the server and thus fit better in _base_ scope
-than the _vendor_ one.
+Section <<required-metrics#required-metrics>> lists the required metrics. This list includes a few items marked as optional. The RESTful metrics are listed as optional as they are expected to be portable between different implementations and thus fit better in the _base_ scope. If the implementation provides RESTful metrics they are enabled by default and can be disabled by setting the MicroProfile Config value `mp.metrics.enable.restful.stats` to `false`. The other metrics are listed as optional as they are dependent on the underlying JVM and not the server and thus fit better in _base_ scope than the _vendor_ one.
```

Comment:
In my opinion it would be better to disable them by default. Not everyone wants their REST endpoints instrumented with all the overhead that it comes with, and those who don't want REST metrics would be forced to add new configuration.
In Quarkus we chose the strategy that if an extension provides its own metrics (data sources, vert.x, fault tolerance,...) they are disabled by default, otherwise a microservice could easily become not-so-micro without the developer even noticing it until it is deployed into production.